src/applications/healthcare/questionnaire/routes.jsx

```diff
@@ -0,0 +1,13 @@
+import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
+import formConfig from './config/form';
+import QuestionnaireWrapper from './containers/QuestionnaireWrapper.jsx';
+
+const route = {
+  path: '/',
+  component: QuestionnaireWrapper,
+  indexRoute: { onEnter: (nextState, replace) => replace('/introduction') },
```

Comment:
Just a heads up you will need to add a redirect via the `devops` repo otherwise if someone sends a user a URL with `/introduction` on it will 404. A problem I ran into.

Reply:
Can you link the docs for that?

Reply:
I am not sure if there are docs for this to be honest, I got this info from Public Websites.